src/app/page/Request/foodbox/ConfirmStep.jsx

```diff
@@ -88,8 +89,10 @@ function ConfirmStep({ dispatch, state }) {
     }
     try {
       const createdMission = await Mission.create(mission);
-      const redirect = isDonationRequest ? "donation" : "payment";
-      history.push(`/request/foodbox/success/${redirect}`);
+      const redirect = isDonationRequest
+        ? routes.request.success.donation
+        : routes.request.success.payment;
+      history.push(redirect);
```

Comment:
Is there a benefit to history.push vs using react routers `Redirect` ?

Reply:
they're essentially the same `Redirect` just offers the "react-y" way to do it, but it still calls the history object behind the scenes. Though I suppose this should be `history.replace` to replicate exactly